### ISSUE TITLE
The "PeriodInSuiteName" Rule does not run on directory names

### ISSUE THREAD

**Issue by bkhouri** [boakley/robotframework-lint#57]
A test suite can be a folder, and running the "PeriodInSuiteName" does not get triggered on the a folder.

Reproducible: 100%

Steps to Reproduce:

1. Create  the following directory structure

    ```
    ├── RFLintTest
    │   ├── MyLinter.Testing.robot
    │   └── SubSuite.WithPeriod
    │       ├── Test.robot
    │       └── __init__.robot
    ```

1. Run `rflint --ignore all --error PeriodInSuiteName --recursive ./RFLintTest` 

    ```
    $ rflint --ignore all --error PeriodInSuiteName --recursive ./RFLintTest/
    + ./tests/RFLintTest/MyLinter.Testing.robot
    E: 0, 0: '.' in suite name 'MyLinter.Testing' (PeriodInSuiteName)
    ```
